src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -142,7 +142,7 @@
     - name: Deploy source code to an app. This will pack current directory, build binary with Pivotal Build Service and then deploy to the app.
       text: az spring-cloud app deploy -n MyApp -s MyCluster -g MyResourceGroup
     - name: Deploy a pre-built jar to an app with jvm options and environment variables.
-      text: az spring-cloud app deploy -n MyApp -s MyCluster -g MyResourceGroup --jar-path app.jar --jvm-options="-XX:+UseG1GC -XX:+UseStringDeduplication" --env foo=bar
+      text: az spring-cloud app deploy -n MyApp -s MyCluster -g MyResourceGroup --artifact-path app.jar --jvm-options="-XX:+UseG1GC -XX:+UseStringDeduplication" --env foo=bar

```

Comment:
is --jar-path a ga parameter? If yes, pls follow the guideline to deprecate it https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments. Remove a GAed param will break existing users.

Reply:
That's right, and we'd like to deprecate it. I've followed the guideline to redirect it. See below message:

```
Option '--jar-path' has been deprecated and will be removed in a future release. Use '--artifact-path' instead.
This command usually takes minutes to run. Add '--verbose' parameter if needed.
file_type is NetCoreZip
[1/3] Requesting for upload URL
[2/3] Uploading package to blob
[3/3] Updating deployment in app 'planet-weather-provider' (this operation can take a while to complete)
```